### PR TITLE
[Bulk] Validation: fix missing descriptions

### PR DESCRIPTION
**Pull request by ccard587**
This PR fixes about 20 missing descriptions.

### CODE REVIEW

File: iis/web-development-reference/native-code-api-reference/chttpmodule-onupdaterequestcache-method.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "CHttpModule::OnUpdateRequestCache Method"
 ms.date: "10/07/2016"
+description: This represents the method that will handle an Update Request Cache event, which occurs when IIS stores the request in the cache.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/chttpmodule-onupdaterequestcache-method.md:4]: Let's add the API (methods, types, properties) to the description.  Maybe something like "The OnUpdateRequestCache method will handle an Update Request Cache event, which occurs when IIS stores the request in the cache."

File: iis/web-development-reference/native-code-api-reference/iapphostcollectionschema-clearelementschema-property.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IAppHostCollectionSchema::ClearElementSchema Property"
 ms.date: "10/07/2016"
+description: The IApp Host Collection Schema Clear Element Schema Property gets the schema for the element that will be used to clear all items from the collection.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/iapphostcollectionschema-clearelementschema-property.md:4]: Let's combine the words to reflect the API (property), like this "The IAppHostCollectionSchema ClearElementSchema property gets the schema for the element that will be used to clear all items from the collection."

File: iis/web-development-reference/native-code-api-reference/iapphostelementschema-iscollectiondefault-property.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IAppHostElementSchema::IsCollectionDefault Property"
 ms.date: "10/07/2016"
+description: The IAppHost Element Schema is the Collection Default Property that gets a corresponding IApp Host Element Interface object.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/iapphostelementschema-iscollectiondefault-property.md:4]: This one's tough.  Let's combine the words here too, "The IAppHostElementSchema IsCollectionDefaultProperty gets a corresponding IAppHostElement Interface object.

File: iis/web-development-reference/native-code-api-reference/iapphostelementschema-name-property.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IAppHostElementSchema::Name Property"
 ms.date: "10/07/2016"
+description: The IApp Host Element Schema Name Property gets the name of the current schema.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/iapphostelementschema-name-property.md:4]: Let's combine the words here too, "The IAppHostElementSchema name property gets the name of the current schema."

File: iis/web-development-reference/native-code-api-reference/ihttpapplication-getapplicationid-method.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpApplication::GetApplicationId Method"
 ms.date: "10/07/2016"
+description: The Get Application Id method returns the unique identity path for the application that is processing the current request.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttpapplication-getapplicationid-method.md:4]: How about, "The GetApplicationId method returns the unique identity path for the application that is processing the current request."

File: iis/web-development-reference/native-code-api-reference/ihttpcompletioninfo-getcompletionbytes-method.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpCompletionInfo::GetCompletionBytes Method"
 ms.date: "10/07/2016"
+description: The IHttp Completion Info, Get Completion Bytes Method, returns the number of bytes completed for an asynchronous operation.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttpcompletioninfo-getcompletionbytes-method.md:4]: How about, "The IHttpCompletionInfo, GetCompletionBytes method returns the number of bytes completed for an asynchronous operation."

File: iis/web-development-reference/native-code-api-reference/ihttpconnectionstoredcontext-interface.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpConnectionStoredContext Interface"
 ms.date: "10/07/2016"
+description: IHttp Connection Stored Context Interface defines context information for connection-stored data which exposes the method Notify Disconnect.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttpconnectionstoredcontext-interface.md:4]: Let's do "IHttpConnectionStoredContext interface defines context information for connection-stored data."

File: iis/web-development-reference/native-code-api-reference/ihttpcontext-gettracecontext-method.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpContext::GetTraceContext Method"
 ms.date: "10/07/2016"
+description: IHttp Context Get Trace Context Method retrieves the trace context container for the current context.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttpcontext-gettracecontext-method.md:4]: How about, "IHttpContext GetTraceContext method retrieves the trace context container for the current context."

File: iis/web-development-reference/native-code-api-reference/ihttptokenentry-getprimarytoken-method.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpTokenEntry::GetPrimaryToken Method"
 ms.date: "10/07/2016"
+description: IHttp Token Entry Get Primary Token Method returns the primary token for the process that is servicing a request.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttptokenentry-getprimarytoken-method.md:4]: How about, "IHttpTokenEntry GetPrimaryToken method returns the primary token for the process that is servicing a request."

File: iis/web-development-reference/native-code-api-reference/ihttpuser-getuservariable-method.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IHttpUser::GetUserVariable Method"
 ms.date: "10/07/2016"
+description: The IHttp User Get User Variable Method returns the custom user data by name with a void pointer that contains the upcast data.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ihttpuser-getuservariable-method.md:4]: Let's combine the APIs here too, "The IHttpUser GetUserVariable method returns the custom user data by name with a void pointer that contains the upcast data.

File: iis/web-development-reference/native-code-api-reference/imetadatainfo-getvrtoken-method.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IMetadataInfo::GetVrToken Method"
 ms.date: "10/07/2016"
+description: The IMetadata Info Get Vr Token Method retrieves an IHttp Token Entry interface for the impersonation account that is configured for a virtual directory.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/imetadatainfo-getvrtoken-method.md:4]: "The IMetadataInfo GetVrToken method retrieves an IHttpTokenEntry interface for the impersonation account that is configured for a virtual directory."

File: iis/web-development-reference/native-code-api-reference/imoduleallocator-allocatememory-method.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IModuleAllocator::AllocateMemory Method"
 ms.date: "10/07/2016"
+description: With IModule Allocator, the Allocate Memory Method allocates a specified amount of memory that is specified by the cbAllocation parameter.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/imoduleallocator-allocatememory-method.md:4]: "With IModuleAllocator, the AllocateMemory method allocates a specified amount of memory that is specified by the cbAllocation parameter."

File: iis/web-development-reference/native-code-api-reference/imoduleallocator-interface.md [hunk 13]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IModuleAllocator Interface"
 ms.date: "10/07/2016"
+description: The IModule Allocator Interface allocates memory within the current request's memory pool.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/imoduleallocator-interface.md:4]: "The IModuleAllocator interface allocates memory within the current request's memory pool."

File: iis/web-development-reference/native-code-api-reference/inativeconfigurationelementcollection-deleterawcollectionelement-method.md [hunk 14]
```diff
@@ -1,6 +1,7 @@
 ---
 title: INativeConfigurationElementCollection::DeleteRawCollectionElement Method [IIS 7]
 TOCTitle: INativeConfigurationElementCollection::DeleteRawCollectionElement Method
+description: It deletes a configuration element from a collection of native configuration elements with an HRESULT.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/inativeconfigurationelementcollection-deleterawcollectionelement-method.md:4]: Let's add the API to the description.  "DeleteRawCollectionElement method deletes a configuration element from a collection of native configuration elements.

File: iis/web-development-reference/native-code-api-reference/ipmhealthandidlemonitor-interface.md [hunk 15]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IPmHealthAndIdleMonitor Interface"
 ms.date: "10/07/2016"
+description: This Enables the worker process framework to check whether a protocol manager is healthy, or whether the listener channels for the protocol manager are idle.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ipmhealthandidlemonitor-interface.md:4]: Let's add the API here too, "The IPmHealthAndIdleMonitor interface enables the worker process framework to check whether a protocol manager is healthy, or whether the listener channels for the protocol manager are idle."

File: iis/web-development-reference/native-code-api-reference/iprotocolmanager-interface.md [hunk 16]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IProtocolManager Interface"
 ms.date: "10/07/2016"
+description: This defines the base interface for protocol managers, and exposes methods for the worker process framework to interact with protocol managers.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/iprotocolmanager-interface.md:4]: "The IProtocolManager interface defines the base interface for protocol managers and exposes methods for the worker process framework to interact with protocol managers."

File: iis/web-development-reference/native-code-api-reference/ireadentityprovider-setentity-method.md [hunk 17]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "IReadEntityProvider::SetEntity Method"
 ms.date: "10/07/2016"
+description: The Set Entity method replaces the preloaded HTTP request entity body with the entity body that is pointed to by the pBuffer parameter.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/ireadentityprovider-setentity-method.md:4]: "The SetEntity method replaces the preloaded HTTP request entity body with the entity body that is pointed to by the pBuffer parameter."

File: iis/web-development-reference/native-code-api-reference/isendresponseprovider-setlogdata-method.md [hunk 18]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "ISendResponseProvider::SetLogData Method"
 ms.date: "10/07/2016"
+description: ISend Response Provider Set Log Data Method configures logging information for the current response.
```
Comment (RedZephyr13) [inline iis/web-development-reference/native-code-api-reference/isendresponseprovider-setlogdata-method.md:4]: "ISendResponseProvider SetLogData method configures logging information for the current response."

**Comment by ccard587** [comment]
Global effort to fix build validation errors

@Rick-Anderson - MSFT NOTE: The validation warnings and suggestions in this PR will be fixed by another author in another PR.

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**